### ISSUE TITLE
🛠️ How to Participate in CampusShare Project Work

### ISSUE THREAD

**Issue by mmuhunn** [mmuhunn/CampusShare#17]
## 🛠️ How to Participate in CampusShare Project Work
Welcome to the CampusShare Project! We’re excited to have you on board. Follow the steps below to ensure a smooth and collaborative workflow:

### Switch to the Appropriate Branch
The source code is divided into Campusshare_Front and Campusshare_Back repositories. Make sure to switch to the correct branch before you start working.

Commands to Check and Switch Branch:

# List all branches
```git branch -a  ```

# Switch to the desired branch
```git checkout <branch-name>```

### Verify Remote Repository
Ensure your local repository is connected to the correct remote repository.

* Command to Verify Remote URL: *
```git remote -v```

* If the URL is missing or incorrect, link it to the official repository:
```git remote add origin https://github.com/mmuhunn/CampusShare.git```

* Pull the Latest Changes
Before starting any work, pull the latest updates from the repository to avoid conflicts.

* Command to Pull Changes:
```git pull origin <branch-name>```

* Commit and Push Your Changes
After completing your work, commit your changes locally and push them to the remote repository.

* Commands for Commit and Push:
# Stage changes
```git add .  ```

# Commit changes with a message
```git commit -m "Brief description of changes"```

# Push changes to the repository
``` git push origin <branch-name> ```

Submit Pull Request
Once you’ve pushed your changes, create a pull request for review. If you agree with the proposed changes in another pull request, leave a comment to document your approval.

### Example Comment for Agreement:
I agree with these changes. Good to merge! 👍

Feel free to ask any questions in this Issue thread. Happy coding! 🚀

**Comment by hellinii** [comment]
I agree with the proposed workflow. Looking forward to collaborating! 👍👍

**Comment by protruser** [comment]
Hello

**Comment by Hyunmin0502** [comment]
Hello protruser

**Comment by mmuhunn** [comment]
Welcome, protruser! I anticipate your contributions 👍 😎 👍
